### PR TITLE
[Update] Japanese translation of docs/newbs_building_firmware_configurator.md

### PR DESCRIPTION
**Pull request by mtei**
## Description

Following the original changes, docs/ja/newbs_building_firmware_configurator.md  was divided into docs/ja/configurator_step_by_step.md and docs/ja/configurator_troubleshooting.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/configurator_step_by_step.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# QMK Configurator: Step by Step
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:1]:
```suggestion
# QMK Configurator: ステップ・バイ・ステップ
```

File: docs/ja/configurator_step_by_step.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ドロップダウンボックスをクリックして、キーマップを作成するキーボードを選択します。
+
+?> **キーボードに複数のバージョンがある場合は、正しいバージョンを選択してください。**
+
+大事なことなのでもう一度言います。
+
+!> **正しいバージョンを選択してください!**
+
+キーボードが QMK を搭載していると宣伝されていてもリストにない場合は、開発者がまだ作業中か、私たちがまだマージするきっかけがなかった可能性があります。
+アクティブな [Pull Request](https://github.com/qmk/qmk_firmware/pulls?q=is%3Aopen+is%3Apr+label%3Akeyboard) がない場合、[qmk_firmware](https://github.com/qmk/qmk_firmware/issues)で報告して、その特定のキーボードのサポートをリクエストします。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:22]:
```suggestion
アクティブな [プルリクエスト](https://github.com/qmk/qmk_firmware/pulls?q=is%3Aopen+is%3Apr+label%3Akeyboard) がない場合、[qmk_firmware](https://github.com/qmk/qmk_firmware/issues)で報告して、その特定のキーボードのサポートをリクエストします。
```

File: docs/ja/configurator_step_by_step.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+製作者自身の GitHub アカウントにある QMK 搭載キーボードもあります。
+それも再確認してください。
+
+## ステップ2: キーボードのレイアウトを選択する
+
+作成したいと思うキーマップに最も近いレイアウトを選択します。一部のキーボードには、まだ十分なレイアウトまたは正しいレイアウトが定義されていません。これらは将来サポートされる予定です。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:28]:
```suggestion
作成したいと思うキーマップに最も近いレイアウトを選択します。一部のキーボードには、まだ十分なレイアウトyや正しいレイアウトが定義されていません。これらは将来サポートされる予定です。
```

File: docs/ja/configurator_step_by_step.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## ステップ3: キーマップの名前を決める
+
+お好みの名前をキーマップにつけます。
+
+?> コンパイル時に問題が発生した場合は、もしかすると QMK ファームウェアリポジトリに既に同じ名前が存在しているのかもしれません、名前を変更してみてください。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:34]:
```suggestion
?> コンパイル時に問題が発生した場合は、もしかすると QMK ファームウェアリポジトリに既に同じ名前が存在しているのかもしれません。名前を変更してみてください。
```

File: docs/ja/configurator_step_by_step.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+## ステップ4: キーマップを定義する
+
+キーコードの入力は、3つの方法のいずれかで行います。
+
+1. ドラッグ・アンド・ドロップ
+2. レイアウト上の空の場所をクリックして、希望するキーコードをクリックします
+3. レイアウト上の空の場所をクリックして、キーボードの物理キーを押します
+
+?> マウスをキーの上に置くと、そのキーコードの機能の短い説明文が出ます。より詳細な説明については以下を見てください。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:44]:
```suggestion
?> マウスをキーの上に置くと、そのキーコードの機能の短い説明文が出ます。より詳細な説明については以下を見てください:
```

File: docs/ja/configurator_step_by_step.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+* [Basic Keycode Reference](ja/keycodes_basic.md)
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:46]:
```suggestion
* [基本的なキーコードリファレンス](ja/keycodes_basic.md)
```

File: docs/ja/configurator_step_by_step.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+!> 選択したレイアウトが物理的なビルドと一致しない場合は、使用していないキーは空白のままにしておきます。どのキーが使用されているかわからない場合、例えば、バックスペースキーは 1 つだが `LAYOUT_all` には2つのキーがあるなど、同じキーコードを両方の場所に配置してください。
+
+## ステップ5: 後日のためにキーマップを保存する
+
+キーマップに満足するか、または後で作業したい場合は、`Export Keymap' ボタンを押します。
+これでキーマップがあなたのコンピュータに保存されます。
+その後、`Import Keymap` ボタンを押すことで、この .json ファイルを後で読み込むことができます。
+
+!> **注意:** このファイルは、kbfirmware.com またはその他のツールに使用される .json ファイルと同じ形式ではありません。これらのツールにこれを使用したり、QMK Configurator でこれらのツールの .json を使用しようとすると、問題が発生します。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:57]:
```suggestion
!> **注意:** このファイルは、kbfirmware.com またはその他のツールに使用される .json ファイルと同じ形式ではありません。これらのツールにこの .json を使用したり、QMK Configurator でこれらのツールの .json を使用しようとすると、問題が発生します。
```

File: docs/ja/configurator_step_by_step.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+1. ドラッグ・アンド・ドロップ
+2. レイアウト上の空の場所をクリックして、希望するキーコードをクリックします
+3. レイアウト上の空の場所をクリックして、キーボードの物理キーを押します
+
+?> マウスをキーの上に置くと、そのキーコードの機能の短い説明文が出ます。より詳細な説明については以下を見てください。
+
+* [Basic Keycode Reference](ja/keycodes_basic.md)
+* [Advanced Keycode Reference](ja/feature_advanced_keycodes.md)
+
+!> 選択したレイアウトが物理的なビルドと一致しない場合は、使用していないキーは空白のままにしておきます。どのキーが使用されているかわからない場合、例えば、バックスペースキーは 1 つだが `LAYOUT_all` には2つのキーがあるなど、同じキーコードを両方の場所に配置してください。
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:49]:
```suggestion
!> 選択したレイアウトが物理的なビルドと一致しない場合は、使用していないキーは空白のままにしておきます。どのキーが使用されているかわからない場合、例えば、バックスペースキーは1つだが `LAYOUT_all` には2つのキーがある場合は、同じキーコードを両方の場所に配置してください。
```

File: docs/ja/configurator_step_by_step.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+キーコードの入力は、3つの方法のいずれかで行います。
+
+1. ドラッグ・アンド・ドロップ
+2. レイアウト上の空の場所をクリックして、希望するキーコードをクリックします
+3. レイアウト上の空の場所をクリックして、キーボードの物理キーを押します
+
+?> マウスをキーの上に置くと、そのキーコードの機能の短い説明文が出ます。より詳細な説明については以下を見てください。
+
+* [Basic Keycode Reference](ja/keycodes_basic.md)
+* [Advanced Keycode Reference](ja/feature_advanced_keycodes.md)
```
Comment (shelaf) [inline docs/ja/configurator_step_by_step.md:47]:
```suggestion
* [高度なキーコードリファレンス](ja/feature_advanced_keycodes.md)
```

File: docs/ja/configurator_troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+## 私の .json ファイルが動きません
+
+.json ファイルが QMK Configurator で作ったものの場合、おめでとうございます。バグに遭遇しました。 [qmk_configurator](https://github.com/qmk/qmk_configurator/issues) で報告してください。
+
+そうでない場合は、... 他の .json ファイルを使用しないようにという、上に書いた注意書きを見逃してませんか？
+
+#### レイアウトに余分なスペースがありますか？どうすればいいですか？
+
+もしスペースバーが3つに分かれている場合は、全てスペースバーで埋めるのが最善の方法です。バックスペースやシフトキーについても同じことができます。
```
Comment (shelaf) [inline docs/ja/configurator_troubleshooting.md:17]:
```suggestion
もしスペースバーが3つに分かれている場合は、全てスペースバーで埋めるのが最善の方法です。バックスペースや Shift キーについても同じことができます。
```

File: docs/ja/configurator_troubleshooting.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+#### キーコードってなに？
+
+以下を見てください。
+
+* [Basic Keycode Reference](ja/keycodes_basic.md)
```
Comment (shelaf) [inline docs/ja/configurator_troubleshooting.md:23]:
```suggestion
* [基本的なキーコードリファレンス](ja/keycodes_basic.md)
```

File: docs/ja/configurator_troubleshooting.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+* [Advanced Keycode Reference](ja/feature_advanced_keycodes.md)
```
Comment (shelaf) [inline docs/ja/configurator_troubleshooting.md:24]:
```suggestion
* [高度なキーコードリファレンス](ja/feature_advanced_keycodes.md)
```

File: docs/ja/configurator_troubleshooting.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+#### コンパイルできません
+
+キーマップの他のレイヤーを再確認して、ランダムなキーが存在しないことを確認してください。
```
Comment (shelaf) [inline docs/ja/configurator_troubleshooting.md:28]:
```suggestion
キーマップの他のレイヤーを再確認して、おかしなキーが存在しないことを確認してください。
```

File: docs/ja/newbs_building_firmware_configurator.md [hunk 14]
```diff
@@ -2,112 +2,17 @@
...
+  git diff 0.9.0 HEAD -- docs/newbs_building_firmware_configurator.md | cat
 -->
 
-[QMK Configurator](https://config.qmk.fm) は、QMKファームウェアの hex ファイルを生成するオンライングラフィカルユーザーインターフェイスです。
+[![QMK Configurator Screenshot](https://i.imgur.com/anw9cOL.png)](https://config.qmk.fm/)
 
-?> **次の手順を順番に実行してください。**
+[QMK Configurator](https://config.qmk.fm) は、QMKファームウェアの hex ファイルを生成するオンライングラフィカルユーザーインターフェイスです。
 
 [Video Tutorial](https://www.youtube.com/watch?v=-imgglzDMdY) を見てください。
```
Comment (shelaf) [inline docs/ja/newbs_building_firmware_configurator.md:13]:
```suggestion
[ビデオチュートリアル](https://www.youtube.com/watch?v=-imgglzDMdY) を見てください。
```

File: docs/ja/newbs_building_firmware_configurator.md [hunk 15]
```diff
@@ -2,112 +2,17 @@
...
+多くの人は、それが自分のキーボードのプログラミングを始めるのに十分な情報であることに気づくでしょう。
 
 QMK Configurator は Chrome/Firefox で最適に動作します。
 
 !> **KLE や kbfirmware などの他のツールのファイルは、QMK Configurator と互換性がありません。それらをロードしたり、インポートしたりしないでください。QMK Configurator は異なるツールです。**
```
Comment (shelaf) [inline docs/ja/newbs_building_firmware_configurator.md:18]:
```suggestion
!> **注意: Keyboard Layout Editor (KLE) や kbfirmware などの他のツールのファイルは、QMK Configurator と互換性がありません。それらをロードしたり、インポートしたりしないでください。QMK Configurator は異なるツールです。**

[QMK Configurator: ステップ・バイ・ステップ](ja/configurator_step_by_step.md)を参照してください。
```

**Comment by mtei** [comment]
> 原文15行目の `<!---` `-->` 間にあるコメント文はどうしますか？

無視しようと思います。コメントの通りに原文のそこの段落が改訂されたら、和訳も直せばいいと思います。

File: docs/ja/configurator_step_by_step.md (hunk 4 above)
Comment (mtei) [reply docs/ja/configurator_step_by_step.md:34]: 原文が、1文なので、以下はどうでしょう？
```suggestion
?> コンパイル時に問題が発生した場合は、QMK ファームウェアリポジトリに既に名前が存在している可能性があるので、この名前を変更するとよいかもしれません。
```

**Pull request merged**